src/sif/autocost.cc

```diff
@@ -463,6 +476,8 @@ Cost AutoCost::TransitionCost(const baldr::DirectedEdge* edge,
       turn_cost += 1.5f;
       if (edge->roundabout())
         turn_cost += 0.5f;
+    } else if (edge->use() == Use::kAlley && pred.use() == Use::kAlley) {
+      turn_cost += 1.5f; // add a cost to transition across alleys
```

Comment:
Nit: should we pull this 1.5f into a constant? I'm noticing we have other hardcoded costs on the lines above, but it would help with readability. Up to you.

Reply:
@danpaz I agree.  In fact, we should do for all the costings at some point.  For now though I will let it go, if that is ok

Reply:
@gknisely yup fine to defer to another PR

Reply:
Opened https://github.com/valhalla/valhalla/issues/2232 for tracking.